catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsSource.java

```diff
@@ -416,9 +418,15 @@ private WFSCapabilitiesType getCapabilities() throws SecurityServiceException {
         try {
             capabilities = wfs.getCapabilities(new GetCapabilitiesRequest());
         } catch (WfsException wfse) {
-            LOGGER.warn(WFS_ERROR_MESSAGE, wfse);
+            LOGGER.warn(WFS_ERROR_MESSAGE
+                            + " Received HTTP code '{}' from server for source with id='{}'",
+                    wfse.getHttpStatus(),
+                    getId());
+            LOGGER.debug(WFS_ERROR_MESSAGE, wfse);
```

Comment:
When the source gets back the CswException from CswResponseExceptionMapper, the source should take care of adding its ID to the log message.